client/signup/main.jsx

```diff
@@ -667,9 +664,16 @@ class Signup extends Component {
 			};
 		}
 
+		const classes = [ 'signup__step' ];
+		if ( isP2Flow( this.props.flowName ) ) {
+			classes.push( `is-${ this.props.stepName }` );
+		} else {
+			classes.push( `is-${ kebabCase( this.props.stepName ) }` );
+		}
+
 		return (
 			<div className="signup__step" key={ stepKey }>
-				<div className={ `signup__step is-${ kebabCase( this.props.stepName ) }` }>
```

Comment:
Why do we need to make an exception for the P2 signup flow here and below?

Reply:
Mmyeah not too sold on doing this, but `kebabCase` was making the classname `is-p-2`/`is-p-2-v-2` (instead of `is-p2`/`is-p2v2`), which I found confusing at first. But what do you think? Is it worth the extra logic just for a "proper-looking" classname?

Reply:
Ah, I see. I'm thinking whether it makes sense to call the flow `p2v2`. Maybe we could do something like `p2-new`? I suppose anything that would satisfy the original `kebabCase` so we don't have to refactor it. 😅  Since we'll be changing the name to `p2` after it's done anyways and I'd say `p2v2` is not user friendly url part anyways.

Reply:
Changed the flowname to `p2-new` (that was its original name! 😅). Some of the classnames became, e.g. `is-p-2-new`, `is-p-2-get-started`, but I guess that's an acceptable tradeoff compared to having extra classname logic for P2 flows.